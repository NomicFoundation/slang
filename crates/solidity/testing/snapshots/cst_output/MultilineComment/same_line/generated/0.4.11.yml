# This file is generated automatically by infrastructure scripts. Please don't edit by hand.

Source: >
  1  │ /* this is on the same line */                                                   │ 0..30

Errors: []

Tree:
  - MultilineComment (Token): "/* this is on the same line */" # 0..30
