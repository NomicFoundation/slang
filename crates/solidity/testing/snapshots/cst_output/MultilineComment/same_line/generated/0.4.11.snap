# This file is generated automatically by infrastructure scripts. Please don't edit by hand.

Errors: 0

► Token(MultilineComment) - (0..30)     │ /* this is on the same line */
